src/App.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
```

Comment:
I think `react-router-dom` should be installed in this branch rather than having the reviewer install it. That might be why the checks are failing. Netlify doesn't know to install it unless it's already been added to the `package.json`.

Reply:
@skillitzimberg Thanks! This was very helpful and I realized what I did wrong. The `package.json` files are not updated.

Reply:
Did you `npm install react-router-dom` @SaraSweetie to install the library?

Reply:
@stevelikesmusic Yes. That is how I installed the react-router-dom library?